src/llmcompressor/utils/helpers.py

```diff
@@ -974,7 +974,8 @@ def getattr_chain(obj: Any, chain_str: str, *args, **kwargs) -> Any:
     return res
 
 
-class DisableKVCache:
+@contextlib.contextmanager
+def disable_cache(module: torch.nn.Module):
```

Comment:
definitely agree with these changes, but might be better in a separate PR or added in the PR summary. Seems orthogonal to `calibrate_all_experts`

Reply:
This was prompted by a type-hint change. I think this should be land in this PR